5.machine_learning/1.evaluate_models/scripts/3.ROC_QC_new_method.py

```diff
@@ -0,0 +1,245 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# ## Perform bootstrapping ROC AUC method to determine if QC significantly improves performance of model
```

Comment:
We will benefit in the future from doing one of two things (maybe both).

1. Clarify what you mean by "new" in the docstring - what precisely is different?
2. Change the name of the file to be more descriptive of what you mean by "new"

Reply:
I'm not sure if you addresed this?

Reply:
I hadn't addressed any of the comments yet, haha. I was looking to chat IRL but I will try and address and we can chat later.

Reply:
Done! Let me know your thoughts on the changes!